src/assets/locales/en.js

```diff
@@ -30,7 +30,7 @@ export default {
     headings: {
       add_question: "Add a new question",
       subjective_question_warning:
-        "Subjective Questions are not included while calculating the accuracy in the dashboard",
+        "All answers to subjective questions are considered correct",
```

Comment:
<img width="654" alt="Screenshot 2021-09-27 at 12 10 03 PM" src="https://user-images.githubusercontent.com/13185699/134857021-4bca4850-9e10-4fc0-a99c-21be59b1a1b8.png">
We'll need to center the text.